operator/v1/types_console.go

```diff
@@ -21,13 +21,27 @@ type ConsoleSpec struct {
 	// customization is used to optionally provide a small set of
 	// customization options to the web console.
 	// +optional
-	Customization ConsoleCustomization `json:"customization"`
+	Customization ConsoleCustomization `json:"customization,omitempty"`
+	// providers contains configuration for using specific service providers.
+	Providers ConsoleProviders `json:"providers,omitempty"`
 }
 
 type ConsoleStatus struct {
 	OperatorStatus `json:",inline"`
 }
 
+type ConsoleProviders struct {
+	// statuspage contains ID for statuspage.io page that provides status info about.
+	// +optional
+	Statuspage *StatuspageProvider `json:"statuspage,omitempty"`
+}
+
+// StatuspageProvider provides identity for statuspage account.
```

Comment:
`StatusPageProvider`, each word has to start with a capital letter.

Reply:
`Statuspage` is a single word due to the brand of the product we are supporting.  I thought the same thing at first, but it's not like GitHub.  [see here](https://www.statuspage.io/)